puppet/modules/web/manifests/init.pp

```diff
@@ -17,8 +17,6 @@
 ) {
   include web::base
   include rsync::server
-  include apache::mod::expires
-  include apache::mod::headers
```

Comment:
Headers is not included by default (only on freebsd). The expires one caught me off guard, but I think without this line it would break.